community/merchants/index.md

```diff
@@ -85,7 +85,25 @@ meta_descr: merchants.descr
           </div>
         </div>
       </div>
-    </div>
+      <div class="right half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
+        <div class="info-block">
+          <div class="center-xs">
+            <img class="merch" src="/img/merchants/basicswapdex.png" alt="BasicSwapDEX Logo" title="BasicSwapDEX">
+            <p>{% t merchants.basicswapdexdesc %}</p>
+          </div>
+          <div>
+            <p><b>{% t merchants.cardfoss %}</b> <span class="cross"></span></p>
+            <p><b>{% t merchants.noncustodial %}</b> <span class="check"></span></p>
+            <p>{% t merchants.cardkyc %} <span class="check"></span></p>
+            <p><b>XMR &#8596; fiat:</b> <span class="cross"></span> {% t merchants.cardbtcfiat %}</p>
+            <p><b>{% t merchants.cardonion %}</b> <span class="check"></span></p>
```

Comment:
```suggestion
            <p><b>{% t merchants.cardfoss %}</b> <span class="check"></span></p>
            <p><b>{% t merchants.noncustodial %}</b> <span class="check"></span></p>
            <p>{% t merchants.cardkyc %} <span class="check"></span></p>
            <p><b>XMR &#8596; fiat:</b> <span class="cross"></span></p>
            <p><b>{% t merchants.cardonion %}</b> <span class="cross"></span></p>
```

i set onion to "no", because its a locally run service. If you want an onion, you can setup and access from an onion. 
the basicswap docker does proxy connections to tor as well, but im not sure any of this earns a check without a note indicating setup required. 

removed btc <> fiat
open source to yes.

Reply:
Thanks. reg the onion address. bisq, which is comparable to basicswapdex has a cross on "onion address" also. but after a brief search i found https://docs.bisq.network/getting-started.html
> Once the installer is finished, go ahead and open Bisq. It’ll take a few moments to open as it connects to the Tor network and syncs with Bisq’s peer-to-peer trading network.

but then bisq has this open issue so im not sure https://github.com/bisq-network/ops/issues/5

clearly, both these service support / allow users to interact with them using tor after some setup and reading setup guides. a check mark with an asterisk with a note to that effect seems sane.

Reply:
i change the suggestion to add asterisk.  Not sure i did it rught